package.json

```diff
@@ -1,77 +1,48 @@
 {
   "name": "lookml-diagram-extension",
+  "sideEffects": [
```

Comment:
Is this the magic word?

Reply:
The css certainly needs to be included in the side effects but there is a bunch of other stuff that has been done to reduce the module size. It's much better but can be improved more I believe. This is good improvement though.
What I would really like to see however is a general cleanup of how CSS is used. Imports of CSS should be replaced with styled components

Reply:
So I saw a separate PR to get rid of the css. At the end of the day sideEffects has an impact on modules but this is not a module. Most likely delete for now